README.md

```diff
@@ -50,238 +50,99 @@ Course authors can fully define and customize the lab environment. It is only
 limited by the feature set of the cloud's deployment features.
 
 
-## Deployment
-
-### Deployment with [Tutor](https://docs.tutor.overhang.io)
+## Deployment with [Tutor](https://docs.tutor.overhang.io)
 
 Running this XBlock with Tutor (for Open edX Maple and later) requires
 two steps:
 
 1. Install the XBlock to your Tutor environment by adding it to the
-  `OPENEDX_EXTRA_PIP_REQUIREMENTS` list in `config.yml`:
-  ```
-  OPENEDX_EXTRA_PIP_REQUIREMENTS:
-    - "hastexo-xblock>=6"
-  ```
-  For additional information, please refer to the [official documentation](https://docs.tutor.overhang.io/configuration.html?highlight=xblock#installing-extra-xblocks-and-requirements)
+   `OPENEDX_EXTRA_PIP_REQUIREMENTS` list in `config.yml`:
+   ```
+   OPENEDX_EXTRA_PIP_REQUIREMENTS:
+     - "hastexo-xblock>=6"
+   ```
+   For additional information, please refer to the [official
+   documentation](https://docs.tutor.overhang.io/configuration.html?highlight=xblock#installing-extra-xblocks-and-requirements).
```

Comment:
Remove the highlighting of the word "xblock" from the docs page:
```suggestion
   documentation](https://docs.tutor.overhang.io/configuration.html#installing-extra-xblocks-and-requirements).
```

Reply:
Yeah, fair point.